src/Tickets/Commerce/Order_Modifiers/Admin/Editor.php

```diff
@@ -85,9 +85,8 @@ public function unregister(): void {
 	 *     selectedFeesByPostId: array<int, string>,
 	 * }
 	 */
-	public function get_store_data(): array {
+	private function get_store_data(): array {
```

Comment:
I see why the tests. But i cant understand why the production code has to be private. We ll need to revert this.